[ADD] localization: philippines-l10n_ph (#5187)
odoo/documentation: content/applications/finance/fiscal_localizations/philippines.rst

A minimum configuration default chart of accounts is installed and the following types of taxes are
installed and also linked to the relevant account:

- VAT 12%
- VAT Exempt
- Withholding taxes

For the withholding taxes, there is an additional :guilabel:`Philippines ATC` field under the
:guilabel:`Philippines` tab.

.. image:: philippines/philippines-atc-code.png
   :alt: Philippines ATC code field set on taxes.

This is the ATC code for that tax when it is exported in the BIR 2307 file and will need to be
filled in manually if the taxes are manually created.

Contacts
--------

If a contact record is set as a :guilabel:`Company` and the :guilabel:`Country` is `Philippines`,
fill in the :guilabel:`Tax ID` field with the `Taxpayer Identification Number (TIN)`.

.. image:: philippines/philippines-contact-company.png
   :alt: Company type contact with Tax ID field.

If a contact record is set as a :guilabel:`Individual`, the :guilabel:`Country` is `Philippines` and
it does not belong to any company, fill in the :guilabel:`Tax ID` field with the `Taxpayer
Identification Number (TIN)`.
There are additional fields for:

- :guilabel:`First Name`
- :guilabel:`Middle Name`

> - :guilabel:`Last Name`

Author: @xpl-odoo I like your shortened version. What's your reasoning for keeping the second screenshot? Looking at it now, I feel like both the company and individual screenshots could be removed.

> - :guilabel:`Last Name`

Reviewer: Just because there are fields that are specific to this localization.
You can remove both, in my opinion. The screenshots repeat what is being said, making them not so useful.

> - :guilabel:`Last Name`

Reviewer: @dade-odoo you mentioned you liked my suggestion and didn't think the screenshots were useful here. Yet this wasn't changed at all? (I mean, I'm ok with that, but you may have forgotten to change this part?)

> - :guilabel:`Last Name`

Author: Good catch :) I decided to keep the single screenshot since it does show fields specific to this localization.

.. image:: philippines/philippines-contact-individual.png
   :alt: Individual type contact with First, Middle, and Last Name fields.

.. note::
   For both :guilabel:`Company` and :guilabel:`Individual`, the TIN would follow the format
   `NNN-NNN-NNN-NNNNN`. The branch code should follow the last digits of the TIN or else it can be
   left as `00000`.

BIR 2307 Report
===============

> **BIR 2307** report data also known as *Certificate of Creditable Tax Withheld at Source*
> can be generated for purchase orders and vendor payments with the applicable withholding taxes.

Reviewer: ```suggestion
**BIR 2307** report data, also known as *Certificate of Creditable Tax Withheld at Source*,
can be generated for purchase orders and vendor payments with the applicable withholding taxes.
```

> The file can be generated from :menuselection:`Vendor Bills --> Action --> Download BIR 2307 XLS`.

Reviewer: ```suggestion
The file can be generated by going to :menuselection:`Accounting --> Vendors --> Bills,` selecting one or multiple vendor bills, and clicking :menuselection:`Action --> Download BIR 2307 XLS`.
```
Or Accounting Dashboard --> Vendor Bills
I feel that when we use :menuselection:, we need to give the full path. But you could go for something mike _on the vendor bills list view, select one or multiple..._

Author: How do you feel about using :menuselection: for the action menu? Does my edit work or would you rather reserve the :menuselection: for the navigation menu?

Reviewer: You can definitely use `:menuselection:` for other menus :)

.. image:: philippines/philippines-bill.png
   :alt: Vendor bill with withholding tax and "download BIR 2307 XLS action.

The same action can be performed on multiple vendor bills.

.. image:: philippines/philippines-multi-bill.png
   :alt: Multiple vendor bills selected with action to "Download BIR 2307 XLS".

A pop-up will appear to review the selection, and then click on :guilabel:`Generate`.

.. image:: philippines/philippines-generate.png
   :alt: Pop up menu to generate BIR 2307 XLS file.

This will generate the `Form_2307.xls` file that lists all the vendor bill lines with the applicable
withholding tax.

.. note::
   If there is no withholding tax applied then the XLS file will not generate records for those
   vendor bill lines.

> The same process above can be done for a *single* vendor :doc:`payment <../accounting/payments>` as
> well if they were linked to one or more :doc:`vendor bills <../accounting/payments/multiple>` with
> applied withholding taxes.

Reviewer: ```suggestion
The same process above can also be done for a *single* vendor :doc:`payment <../accounting/payments>`
if they were linked to one or more :doc:`vendor bills <../accounting/payments/multiple>` with
applied withholding taxes.
```

> .. image:: philippines/philippines-payment.png

Reviewer: Too wide. Plus, users should get it by now where to find this action.

   :alt: Action to Download BIR 2307 XLS from payment

.. note::

>    When grouping payment for multiple bills, Odoo will split the payments based on the contact.
>    When :menuselection:`Payment --> Action --> Download BIR 2307 XLS` the report will only be for
>    those vendor bills for the single contact only.

Reviewer: ```suggestion
   When grouping payments for multiple bills, Odoo splits the payments based on the contact.
   When clicking :menuselection:`Payment --> Action --> Download BIR 2307 XLS`, the report only
   includes vendor bills related to that contact.
```
The last sentence wasn't clear.

.. important::

>    Odoo cannot generate the BIR 2307 PDF report or `DAT` files directly.

Reviewer: ```suggestion
   Odoo cannot generate the BIR 2307 PDF report or DAT files directly.
```
No need for backticks

>    The generated `Form_2307.xls` file can be exported to an *external* tool to convert it to BIR
>    `DAT` format or PDF.

Reviewer: ```suggestion
   The generated `Form_2307.xls` file can be exported to an *external* tool to convert it to the BIR
   DAT or PDF format.
```
